### PR TITLE
Create Parser classes for Articles

### PR DESCRIPTION
**Pull request by Murugan-Maniish**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/FindArticleCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    public FindArticleCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
+        if (trimmedArgs.isEmpty()) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindArticleCommand.MESSAGE_USAGE));
+        }
+
+        String[] nameKeywords = trimmedArgs.split("\\s+");
+
+        return new FindArticleCommand(new NameContainsKeywordsPredicate(Arrays.asList(nameKeywords)));
```
Comment (bennyLCK) [inline src/main/java/seedu/address/logic/parser/FindArticleCommandParser.java:30]: call the constructor for `TitleContainsKeywordsPredicate` instead.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -121,4 +127,97 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
...
+     */
+    public static String parseCategory(String category) throws ParseException {
+        requireNonNull(category);
+        String trimmedCategory = category.trim();
+        //removed the check for category validity
+        return trimmedCategory;
+    }
+
+    /**
+     * Parses a {@code String status} into a {@code Status}.
```
Comment (Ko-Khan) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:207]: Wrong type in documentation

**Review by Ko-Khan — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/EditArticleCommandParser.java
Comment (bennyLCK) [inline src/main/java/seedu/address/logic/parser/EditArticleCommandParser.java:1]: LGTM

**Review by H4mes — APPROVED** [review]
LGTM

**Pull request merged**